### PR TITLE
MAINT, DOC: Update 1.13.0 release notes and .mailmap

### PR DESCRIPTION
**Pull request by charris**
* Update .mailmap for some new contributors.
* First edit of 1.13.0 release notes

### CODE REVIEW

File: doc/release/1.13.0-notes.rst [hunk 1]
```diff
@@ -4,20 +4,31 @@ NumPy 1.13.0 Release Notes
...
+   override default ufunc behavior.
+ * New ``np.block`` function for creating blocked arrays.
+
 
+New functions
+=============
 
-Dropped Support
-===============
+* New ``np.positive`` ufunc provides override for ``__pos__``.
```
Comment (shoyer) [inline doc/release/1.13.0-notes.rst]: This isn't true yet, unfortunately -- see https://github.com/numpy/numpy/pull/8967#issuecomment-299955657

So either we'll want to get that deprecation warning/override in, or change this note

File: doc/release/1.13.0-notes.rst [hunk 2]
```diff
@@ -157,13 +177,61 @@ more detailed description of the implementation and behavior of this new
 option. The API is provisional, we do not yet guarantee backward compatibility
 as modifications may be made pending feedback.
```
Comment (shoyer) [inline doc/release/1.13.0-notes.rst]: Can we add links to the NEP design doc and documentation here?

File: doc/release/1.13.0-notes.rst (hunk 1 above)
Comment (charris) [inline doc/release/1.13.0-notes.rst]: Sounds like that will need more discussion/work. I'll just remove the override bit.

File: doc/release/1.13.0-notes.rst (hunk 2 above)
Comment (charris) [inline doc/release/1.13.0-notes.rst]: I've added some links, but I not sure they are correct :(

**Pull request merged**